string joystickName
int8 joystickNumButtons
int8 joystickNumAxes
int8 joystickNumHats

# Joystick Buttons
int8[] Buttons

# Joystick Sticks
float64[] Axes

# Joystick Hats
int16[] Hats